tree-view-sample-plugin/src/comments.ts

```diff
@@ -0,0 +1,170 @@
+import * as theia from '@theia/plugin';
+
+const comments: Map<string, string[]> = new Map<string, string[]>();
+
+comments.set('Ann', ['Great plugin!', 'Can you share the link to this example?']);
+comments.set('Yevhen', ['Cleanup your code please']);
+comments.set('Sun', ['Do not forget about docs']);
+
+const ON_DID_SELECT_USER = 'comments-tree-on-did-select-user';
+const ON_DID_SELECT_COMMENT = 'comments-tree-on-did-select-comment';
+
+export class Comments {
+
+    treeDataProvider: TestDataProvider;
+    tree: theia.TreeView<string>;
+
+    selectedUser: string | undefined = undefined;
+
+    constructor(public disposables: theia.Disposable[]) {
+        this.treeDataProvider = new TestDataProvider();
+        this.tree = theia.window.createTreeView('comments', { treeDataProvider: this.treeDataProvider });
+
+        this.tree.onDidExpandElement(event => {
+            // handle expanding
+        });
+
+        this.tree.onDidCollapseElement(event => {
+            // handle collapsing
+        });
+
+        theia.commands.registerCommand({
```

Comment:
It would be nice to subscribe all commands to utilization with help theia context: 
https://github.com/eclipse/che-theia-samples/blob/master/diagnostics-plugin/src/diagnostics-plugin-frontend-plugin.ts#L18
https://github.com/eclipse/che-theia-samples/blob/master/diagnostics-plugin/src/diagnostics-plugin-frontend-plugin.ts#L32

Reply:
Done. Thanks!